docs/core/tools/dotnet-test.md

```diff
@@ -42,7 +42,7 @@ dotnet test [-h|--help]
 
 ## Description
 
-The `dotnet test` command is used to execute unit tests in a given project. The `dotnet test` command launches the test runner console application specified for a project. The test runner executes the tests defined for a unit test framework (for example, MSTest, NUnit, or xUnit) and reports the success or failure of each test. The test runner and the unit test library are packaged as NuGet packages and are restored as ordinary dependencies for the project.
+The `dotnet test` command is used to execute unit tests in a given project. The `dotnet test` command launches the test runner console application specified for a project. The test runner executes the tests defined for a unit test framework (for example, MSTest, NUnit, or xUnit) and reports the success or failure of each test. If all tests were successful, then the test runner will return 0 as exit code otherwise it will return 1 if any of the test is failed. The test runner and the unit test library are packaged as NuGet packages and are restored as ordinary dependencies for the project.
```

Comment:
were successful --> are successful (present tense is preferable)
then the test runner will return --> the test runner returns
as exit code --> as an exit code; (note semicolon after "code")
otherwise it will return 1 if any of the test is failed --> otherwise, if any test fails, it returns 1

Reply:
Thanks, @rpetrusha for the review, committed the changes as per comments